BedrockNode.cpp

```diff
@@ -159,7 +177,11 @@ void BedrockNode::_processCommand(SQLite& db, Command* command) {
                  }
              }
             SINFO("Finished upgrading database");
+            _dbReady = true;
```

Comment:
Ah, perhaps `_dbReady` means that it has completed upgrading?  Perhaps `_upgradeComplete` would be more clear.